docs/modules/clients/pages/java.adoc

```diff
@@ -24,14 +69,53 @@ which you may already have done to start using Hazelcast:
 </dependency>
 ----
 
-You can find Hazelcast Java client's code samples https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[here^].
+--
+====
+
+You can find {java-client} code samples in the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[Hazelcast Code Samples repository].
+
+TIP: For a tutorial on getting started with Java in an embedded topology, see xref:getting-started:get-started-java.adoc[].
+
+=== Get started with {java-client-new} (BETA)
+
+To get started using the {java-client-new}, you need to include the `hazelcast-java-client-{full-version}.jar` dependency in your classpath. You can then start using this client as if
+you are using the Hazelcast API. 
+
+If you prefer to use Maven, add the `hazelcast-java-client` dependency to your `pom.xml`:
+
+// check conversation about `hazelcast-enterprise-java-client` dependency and note i.e. is it as below?
```

Comment:
yes, the only difference can be the necessity of adding the jfrog private repository to pom.xml if EE client is used.

Reply:
Not sure if this is documented - can you use **Add a suggestion** to suggest what/where the draft change would be?

Reply:
something like this https://github.com/hazelcast/hz-docs/pull/1289/files#r1761806719